server.py

```diff
@@ -2,13 +2,15 @@
 import os
 import requests
 
-from flask import json, Flask, request, jsonify
+from flask import json, Flask, request, jsonify, redirect
 from flask_jwt_extended import jwt_optional, get_jwt_identity
 
 from qwc_services_core.jwt import jwt_manager
-from qwc_services_core.tenant_handler import TenantHandler
+from qwc_services_core.tenant_handler import TenantHandler, TenantPrefixMiddleware, TenantSessionInterface
 from qwc2_viewer import QWC2Viewer
 
+AUTH_PATH = os.environ.get('AUTH_PATH', '/auth')
+AUTH_REQUIRED = os.environ.get('AUTH_REQUIRED', False)
```

Comment:
We used

     os.environ.get('xxx', '0') not in [0, "0", "False", "FALSE"]

to handle bool env-vars elsewhere.